src/background.js

```diff
@@ -11,25 +11,58 @@ function log() {
 
 const TRR_MODE_PREF = "network.trr.mode";
 
+// This preference is set to TRUE when DoH has been enabled via the add-on. It will
+// allow the add-on to continue to function without the aid of the Normandy-triggered pref
+// of "doh-rollout.enabled". Note that instead of setting it to false, it is cleared.
+const DOH_SELF_ENABLED_PREF = "doh-rollout.self-enabled";
+
+// This pref is set once a migration function has ran, updating local storage items to the
+// new doh-rollot.X namespace. This applies to both `doneFirstRun` and `skipHeuristicsCheck`. 
+const DOH_BALROG_MIGRATION_PREF = "doh-rollout.balrog-migration";
```

Comment:
nit: I'd prefer to be careful and call this "balrog-migration-done" just to be super clear.

Reply:
Updated this request in a seperate PR – #175